src/core/lib/math/dftransform.cpp

```diff
@@ -45,14 +45,29 @@
 namespace lbcrypto {
 
 std::complex<double>* DiscreteFourierTransform::rootOfUnityTable = nullptr;
-size_t DiscreteFourierTransform::m_M                             = 0;
-size_t DiscreteFourierTransform::m_Nh                            = 0;
-bool DiscreteFourierTransform::m_isInitialized                   = false;
+std::unordered_map<uint32_t, DiscreteFourierTransform::PrecomputedValues> DiscreteFourierTransform::precomputedValues;
 
-/// precomputed rotation group indices
-std::vector<uint32_t> DiscreteFourierTransform::m_rotGroup;
-/// precomputed ksi powers
-std::vector<std::complex<double>> DiscreteFourierTransform::m_ksiPows;
+DiscreteFourierTransform::PrecomputedValues::PrecomputedValues(uint32_t m, uint32_t nh) {
```

Comment:
I would change the name of the method from PrecomputedValues to PrecomputeValues

Reply:
It is the constructor for the PrecomputedValues struct.

Reply:
Understood